### PR TITLE
INTMDB-782: Update L1 testing READMEs with correct CFN links

### PR DESCRIPTION
**Pull request by maastha**
## Proposed changes

_Jira ticket:_ [INTMDB-782](https://jira.mongodb.org/browse/INTMDB-782)

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Link to any related issue(s): 

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code
- [ ] I have tested the CDK constructor in a CFN stack. See [TESTING.md](../TESTING.md)

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
Please, make sure to create a Jira ticket and update the title with INTMDB.

File: test/l1-resources/alert-configuration/README.md [hunk 1]
```diff
@@ -7,15 +7,15 @@ This CFN resource must be active in your AWS account while using this constructo
...
+- Follow prerequisite steps for the corresponding [Alert Configuration CFN resource](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/alert-configuration/test/README.md).
 - Set any additional required configuration options/parameters as per your needs.
 - Please follow the steps in [TESTING.md](../../../TESTING.md).
 
 
 ### Success criteria when testing the resource
 1. Alert Settings for the respective Project in Atlas should be correctly configured:
   ![image](https://user-images.githubusercontent.com/5663078/226870968-9ef8ae46-b0cf-462b-ac62-7229d2d79ac0.png)
-2. Ensure general [CDK resource success criteria](../../../TESTING.md#success-criteria-to-be-satisfied-when-testing-a-construct) for this resource is met.
+2. Ensure general [CDK resource success criteria](../../../TESTING.md) for this resource is met.
```
[the remark this answers is not in the document]
Comment (maastha) [reply test/l1-resources/alert-configuration/README.md:18]: This is CDK success criteria (in this repo itself)

**Comment by maastha** [comment]
> Please, make sure to create a Jira ticket and update the title with INTMDB.

Updated

File: test/l1-resources/custom-dns-configuration-cluster-aws/README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ https://cloud-dev.mongodb.com/api/atlas/v1.0/groups/<ATLAS_PROJECT_ID>/awsCustom
 
 ![image](https://user-images.githubusercontent.com/122359335/227661815-d48398a9-aaa3-4978-9de4-736acab6ddf8.png)
 
-2. Ensure general [CDK resource success criteria](../../../TESTING.md#success-criteria-to-be-satisfied-when-testing-a-construct) for this resource is met.
+2. Ensure general [CDK resource success criteria](../../../TESTING.md) for this resource is met.
```
[the remark this answers is not in the document]
Comment (maastha) [reply test/l1-resources/custom-dns-configuration-cluster-aws/README.md:23]: This is CDK success criteria (in this repo itself)

File: test/l1-resources/alert-configuration/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (andreaangiolillo) [reply test/l1-resources/alert-configuration/README.md:18]: https://github.com/mongodb/awscdk-resources-mongodbatlas/tree/main/submodules

**Review by andreaangiolillo — APPROVED** [review]

File: test/l1-resources/alert-configuration/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (maastha) [reply test/l1-resources/alert-configuration/README.md:18]: Oh yes, thanks for sharing that

**Pull request merged**